src/ports/postgres/modules/pmml/test/table_to_pmml.sql_in

```diff
@@ -1,197 +0,0 @@
----------------------------------------------------------------------------
```

Comment:
I deleted src/ports/postgres/modules/pmml/test/table_to_pmml.sql_in and created two files out of it: logistic.sql_in and linear.sql_in